### DISCUSSION TITLE
"'WebView2' could not be found" error after updating to Windows App SDK 1.0 stable

### DISCUSSION THREAD

**Discussion by ilovepumpkin** [MicrosoftEdge/WebView2Feedback#3925] [Q&A]
I was using Windows App SDK 1.0 experimental, and my project is working well. After I upgrade to Windows App SDK 1.0 stable, I got the error "'WebView2' could not be found". Anyone knows what assembly I am missing? 

![image](https://user-images.githubusercontent.com/11310645/144459015-4632fffa-d65e-47f7-b0ee-e2b0dfcf8074.png)

Here is my references
![image](https://user-images.githubusercontent.com/11310645/144459582-1fd9cd85-9e41-46d2-8a0e-f0aac23e6226.png)

**Comment by champnic** [comment]
Are you missing a `using Microsoft.Web.WebView2`? What nuget packages do you have installed? For WindowsAppSdk you don't need the separate WebView2 nuget package.
